Hero-overlay-forest/hero.css

```
@@ -0,0 +1,32 @@
+html,
+body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+
+
+
+}
+
+.forest {
+    width: 100%;
+    height: 100%;
+    background-image: url("https://static.vecteezy.com/system/resources/previews/038/985/779/non_2x/ai-generated-tranquil-scene-of-green-forest-in-summer-generated-by-ai-free-photo.jpg");
+    background-size: cover;
```

As sugera aici sa pui un 
```suggestion
    position:relative
```